src/filter-pattern/pattern-time-relative.ts

```diff
@@ -134,28 +138,28 @@ export const TIME_RELATIVE_PATTERN_DEFINITION: FilterPatternDefinition<TimeRelat
         alignType,
         shiftDuration,
         shiftStep,
+        timezone,
       };
     },
     isValid(_pattern): boolean {
       return true;
     },
-    toExpression(pattern): SqlExpression {
+    toExpression(pattern, timezone?: string): SqlExpression {
```

Comment:
surely timezone should come from the pattern

Reply:
oh of course, duh 😅